ddprof-exporter/src/lib.rs

```diff
@@ -86,12 +91,29 @@ impl ProfileExporterV3 {
         tags: Vec<Tag>,
         endpoint: Endpoint,
     ) -> Result<ProfileExporterV3, Box<dyn Error>> {
-        Ok(Self {
+        let mut exporter = Self {
             exporter: Exporter::new()?,
             endpoint,
             family: family.into(),
             tags,
-        })
+        };
+        exporter.add_useful_tags();
+        Ok(exporter)
+    }
+
+    /// Add some useful tags
+    /// Currently only add Datadog-Container-ID if not already present
+    fn add_useful_tags(&mut self) {
+        let container_id_name = "Datadog-Container-ID";
```

Comment:
This is fine, but I think making this a const would be clearer as later on you are relying on this being `'static`. Maybe `const DATADOG_CONTAINER_ID`? A const can still be in local scope, so don't have to worry about name clashes too much.

Reply:
Good suggestion, I will do that !